iree/compiler/Dialect/Flow/Transforms/TestPartitionableLoopsInterface.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/Flow/IR/PartitionableLoopsInterface.h"
+#include "iree/compiler/Dialect/Flow/Transforms/Passes.h"
+#include "iree/compiler/Dialect/Util/IR/UtilOps.h"
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+
+static const char kAttributeName[] = "__test_interface__";
+
+namespace mlir {
+namespace iree_compiler {
+namespace IREE {
+namespace Flow {
+
+namespace {
+
+/// For ops that implement the `PartitionableLoopsInterface` that have the
+/// `__test_interface__` attribute set generates a `util.unfoldable_constant`
```

Comment:
Could actually just write to `llvm::outs()` and check that, instead of composing the attribute. Might be simpler that way given this is just for testing.